### PR TITLE
Support all securityContext resources

### LINKED ISSUE #786: Support for Security Context for containers in sumologic-kubernetes-collection
As part of Kubernetes' best practices, I'd like to set Security Context values in all the containers deployed on my infrastructure. As of now, the charts in weave-scope lack this capability.

ATM, the only value that get instantiated is the fsGroup:
```
securityContext:
        fsGroup: {{ .Values.fluentd.securityContext.fsGroup }}
```
If we had something like
```
securityContext:
        {{ toYaml .Values.resources | indent 4 }}
```
we could add a full scope of securityContext options to the containers.

### PR DESCRIPTION
**Pull request by astencel-sumo**
Fixes #786

### CODE REVIEW

File: deploy/kubernetes/fluentd-sumologic.yaml.tmpl [hunk 1]
```diff
@@ -878,6 +879,7 @@ spec:
           name: collection-sumologic-fluentd-metrics
       securityContext:
         fsGroup: 999
+
```
Comment (sumo-drosiek) [inline deploy/kubernetes/fluentd-sumologic.yaml.tmpl:882]: We can omit those spaces using `-}}` 🤞

Comment (sumo-drosiek) [reply deploy/kubernetes/fluentd-sumologic.yaml.tmpl:882]: nvm, as we investigate that it's helm2 issue

**Review by sumo-drosiek — APPROVED** [review]
LGTM, please squash all you commits

**Review by perk-sumo — APPROVED** [review]
LGTM 👍

**Review by frankreno — APPROVED** [review]
LGTM!

**Pull request merged**